examples/motion_detection.rs

```diff
@@ -1,7 +1,7 @@
 use mpu6050::{*, device::MOT_DETECT_STATUS};
 use linux_embedded_hal::{I2cdev, Delay};
 use i2cdev::linux::LinuxI2CError;
-use embedded_hal::blocking::delay::DelayMs;
+use embedded_hal::delay::DelayUs;
```

Comment:
Apparently this isn't available everywhere. In my project targeting `embedded-hal` 1.0.0, I get this error:
```
error[E0432]: unresolved import `embedded_hal::delay::DelayUs`
  --> /home/being/.cargo/git/checkouts/mpu6050-25b6f8902ecaa1b9/00d03b4/src/lib.rs:56:5
   |
56 |     delay::DelayUs,
   |     ^^^^^^^-------
   |     |      |
   |     |      help: a similar name exists in the module: `DelayNs`
   |     no `DelayUs` in `delay`
```